end_to_end/signal_handler_test.go

```
@@ -71,9 +104,14 @@ var _ = Describe("Signal handler tests", func() {
 				rng := rand.New(rand.NewSource(time.Now().UnixNano()))
 				time.Sleep(time.Duration(rng.Intn(1000)+500) * time.Millisecond)
 				_ = cmd.Process.Signal(unix.SIGINT)
+
+				for outScanner.Scan() {
+					output.WriteString(outScanner.Text())
+					output.WriteByte('\n')
+				}
 			}()
-			output, _ := cmd.CombinedOutput()
-			stdout := string(output)
+			_ = cmd.Run()
```

IMHO it's bad idea to ignore an error.